src/services/GetGifUrl/getGifUrl.ts

```diff
@@ -0,0 +1,2 @@
+export const getGifUrl = (id: number) =>
+  `https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/versions/generation-v/black-white/animated/${id}.gif`;
```

Comment:
Usually things like this are going to utils / helpers folder. No reason to keep it in services folder